test/e2e/util.go

```diff
@@ -168,7 +168,7 @@ type testEnvironment struct {
 // the desired events or continue to customize the project
 func setupE2ETTestEnvironment(t *testing.T, eventJSONFilePath string, shipyardPath string) testEnvironment {
 	// Just test if we can connect to the cluster
-	clientset, err := keptnutils.GetClientset(false)
+	clientset, err := kubeutils.GetClientSet(false)
 	require.NoError(t, err)
 	assert.NotNil(t, clientset)
```

Comment:
Same here, I don't think we actually need a kubernetes client set for the integration test, but please verify my assumption